request/VoiceRequest.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+public function $rules()
```

Comment:
@debjit I know it's a pseudo-code, but `function rules()` should be without $ sign.

Reply:
Yes! I missed that class/function name can not start with $.